### PR TITLE
Actors page

### PR DESCRIPTION
**Pull request by Adrode**
- Using Grid
- Connected to API
- App.js adjusted to avoid conflicts with actual main branch

### CODE REVIEW

File: src/components/PeopleList/styled.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { styled } from "styled-components";
+
+export const Wrapper = styled.div`
+    display: grid;
```
Comment (mariuszmmm) [inline src/components/PeopleList/styled.js:4]: It's great that you used a grid here.

**Review by mariuszmmm — COMMENTED** [review]
It would be good to place the "PeopleList" component in the section and add a top margin to it. I suggest not changing the size of the space between tiles on smaller screens, but instead adjusting the size of the tiles themselves. Other than that, everything looks very good.

File: src/components/PeopleList/PeopleTile/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+                !poster.includes("null") ?
+                    <ImageContainer>
+                        <Image
+                            src={poster}
+                        />
+                    </ImageContainer>
+
+                    :
+                    <BlankActorContainer>
+                        <BlankActor />
```
Comment (Urbork) [inline src/components/PeopleList/PeopleTile/index.js:24]: Nicely done, it reminded me that i have to resize blank movie poster image

File: src/components/PeopleList/styled.js (hunk 1 above)
Comment (Urbork) [reply src/components/PeopleList/styled.js:4]: Indeed great use of grid

**Review by Urbork — APPROVED** [review]
Amazing job. It is very clear and matches the design

**Pull request merged**